src/MCPClient/lib/clientScripts/email_fail_report.py

```diff
@@ -127,7 +127,7 @@ def get_unit_job_log_html(sip_uuid):
     """
     parser = etree.HTMLParser(remove_blank_text=True)
     rows = (
-        Job.objects.filter(sipuuid=sip_uuid, subjobof="")
```

Comment:
I'm not totally sure of the history of `subjobof`, but as I understand it jobs could previously be nested. It isn't used anymore, but because we have an index on the SIP UUID, but not subjobof, including that clause in the query is much slower.